src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -459,18 +445,17 @@ export class MatAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
    * not adjusted.
    */
   private _scrollToOption(): void {
-    const index = this.autocomplete._keyManager.activeItemIndex || 0;
-    const labelCount = _countGroupLabelsBeforeOption(index,
-        this.autocomplete.options, this.autocomplete.optionGroups);
-
-    const newScrollPosition = _getOptionScrollPosition(
-      index + labelCount,
-      AUTOCOMPLETE_OPTION_HEIGHT,
-      this.autocomplete._getScrollTop(),
-      AUTOCOMPLETE_PANEL_HEIGHT
-    );
-
-    this.autocomplete._setScrollTop(newScrollPosition);
+    const activeItem = this.autocomplete._keyManager.activeItem;
+    if(activeItem && this.autocomplete.panel instanceof ElementRef){
```

Comment:
The formatting seems off here. Also the `panel` should guaranteed to be an `ElementRef` so this check is redundant.